conference.js

```diff
@@ -1288,6 +1288,24 @@ export default {
             APP.UI.updateUserRole(user);
 
             updateRemoteThumbnailsVisibility();
+
+            const currentRemoteUserCount = room.getParticipantCount() - 1;
+            if (currentRemoteUserCount === 1) {
+                const userId = user.getId();
+
+                window.setTimeout(() => {
+                    // Only update if there is one remote user, no one is
+                    // pinned, self is on large, and the remote user is still
+                    // present.
+                    if ((room.getParticipantCount() - 1 === 1)
+                        && !APP.UI.getPinnedId()
+                        && (APP.UI.getLargeVideoID()
+                             === APP.conference.getMyUserId())
+                        && room.getParticipantById(userId)) {
+                        APP.UI.updateLargeVideo(userId);
```

Comment:
I would prefer to do this in VideoLayout.js. A few lines above we call UI.addUser, which eventually ends up in VideoLayout.addParticipantContainer(user). Currently we update the large video in "VideoLayout.remoteVideoActive", so this means that we're going to have all the cases regrouped there.